### DISCUSSION TITLE
Osu lazer files corrupted

### DISCUSSION THREAD

**Discussion by MlerSu** [ppy/osu#22583] [Q&A]
About a month or two ago I was trying to preserve my osu lazer files because I didn't know if lazer itself does that after you uninstall it. I remember the moment it was uninstalling I tried to quickly copy the files, it probably has something to do with that. After I attempted to do that I ended up making my lazer not open in general and even tried to uninstall and reinstall which didn't work. After that I gave up and uninstalled it. 

I think later on within the month or so between then and when I installed lazer again I must've reset it which allowed me to play it again but most, if not all, of my beatmaps have corrupted. most worse than others, it generally ranges from losing storyboard elements, backgrounds, audio, entire beatmaps as well. Even the 3 intros as well as skins were affected as well.

I've tried to and been able to recover a portion of my beatmaps manually but I was wondering if there is (or will be) a way to recover at the very least the intros and beatmaps like how stable does, if that makes sense.

**Comment by bdach** [comment]
Currently this is not possible. It may be at some stage in the future (judging from [this comment](https://github.com/ppy/osu/discussions/22571#discussioncomment-4918815)).
